Sources/Build/Command.compile(SwiftModule).swift

```diff
@@ -34,9 +30,6 @@ extension Command {
         #endif
 
         let tool = SwiftcTool(module: module, prefix: prefix, otherArgs: args + otherArgs, executable: SWIFT_EXEC, conf: conf)
-
-        //FIXME these should be inferred as implicit inputs by llbuild
-        let mkdirs = Set(tool.objects.map{ $0.parentDirectory }).map(Command.createDirectory)
-        return (cmd(tool), mkdirs)
```

Comment:
These probably shouldn't be combined in one commit, this is a subtle semantic change, the other is a trivial refactor.